jiant/proj/main/scripts/configurator.py

```diff
@@ -117,11 +119,7 @@ def create_config(self):
             task_name_list_dict["train_val"] = self.parse_task_name_list(
                 self.train_val_task_name_list
             )
-        full_task_name_list = [
-            task_name
-            for task_name_list in task_name_list_dict.values()
-            for task_name in task_name_list
-        ]
+        full_task_name_list = py_datastructures.get_unique_list_in_order(task_name_list_dict.values())
```

Comment:
`set(task_name_list_dict)` accomplishes the same thing. any motivation for the import?

Reply:
I'm not sure if set has guarantees on the resulting order.

Reply:
Sounds good. Thanks for explaining!